src/main/scala/com/cloudant/clouseau/IndexService.scala

```diff
@@ -117,6 +120,10 @@ class IndexService(ctx: ServiceContext[IndexServiceArgs]) extends Service(ctx) w
         pendingSeq = newSeq
         logger.debug("Pending sequence is now %d".format(newSeq))
         'ok
+      case SetPurgeSeqMsg(newPurgeSeq: Long) =>
+        purgeSeq = newPurgeSeq
```

Comment:
this should be `pendingPurgeSeq = newPurgeSeq`. `purgeSeq` must only change after it's been committed.